test/test_helper.rb

```diff
@@ -2,22 +2,19 @@
 
 unless Rails.env.production?
   require 'simplecov'
-  SimpleCov.start 'rails' do
-    # don't test the schema!
-    # https://github.com/rmosolgo/graphql-ruby/blob/master/guides/schema/
-    # testing.md#dont-test-the-schema
-    add_filter 'db/schema.rb'
-    # these just contain the description of the attributes of the
-    # API responses, no code (https://github.com/Netflix/fast_jsonapi)
-    add_filter 'app/serializers'
-    # Skip testing empty module definition
-    add_filter 'app/models/xccdf.rb'
-    add_filter 'app/services/xccdf_report.rb'
-    # empty class that has to be there for prometheus integration
-    add_filter 'lib/'
+  SimpleCov.start do
+    add_filter 'config'
```

Comment:
It could be beneficial to see coverage on initializers.

Reply:
Unfortunately, it doesn't count initializers even without this filter. All I see in `config` in the coverage report is `routes.rb`. The initializers must already run before the test_helper is required.